### PR TITLE
[Core] GasLimit estimation fixed

### PR DESCRIPTION
**Pull request by flypaper0**

### CODE REVIEW

File: VultisigApp/VultisigApp/Services/BlockChainService.swift [hunk 1]
```diff
@@ -54,25 +54,17 @@ final class BlockChainService {
     
     
     func fetchSpecific(tx: SendTransaction) async throws -> BlockChainSpecific {
-        guard !tx.coin.isNativeToken, tx.coin.chainType == .EVM else {
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Services/BlockChainService.swift:57]: this break none EVM chains

**Review by johnnyluo — CHANGES_REQUESTED** [review]
This cause problem for none EVM chains, please do some tests with 
1. EVM 
2. THORChain
3. UTXO chain 
etc etc.

File: VultisigApp/VultisigApp/Services/BlockChainService.swift (hunk 1 above)
Comment (flypaper0) [reply VultisigApp/VultisigApp/Services/BlockChainService.swift:57]: fixed non evm chains

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**